soci/soci_index.go

```diff
@@ -542,14 +563,7 @@ func GetImageManifestDescriptor(ctx context.Context, cs content.Store, imageTarg
 }
 
 // WriteSociIndex writes the SociIndex manifest to oras `store`.
```

Comment:
Let's go ahead and update this comment while the hood is open on this bit of code. It should match the function name and also state the write is to the blob store.